assets/index.css

```diff
@@ -0,0 +1,1058 @@
+  color: #818181;
+  font-weight: bolder;
+  cursor: pointer;
+}
+
+.subt-turn,
+.subt-culture {
+  font-size: 2vw;
+}
+
+.title-summary a,
+.title-summary-fs a {
+  color: black;
+  line-height: 1px;
+  font-size: 24px;
+  font-weight: bolder;
+}
+
+.image-debate1,
+.image-debate3,
+.image-opinion {
+  width: 10%;
+}
+
+.info-debate11 a,
+.info-debate31 a,
+.info-opinion11 a,
+.info-more-stories11 a {
+  color: black;
+  font-weight: bolder;
+  font-size: 2.2vw;
+}
+
+.deb-line-deco {
+  margin: 2vw 0;
+  border-top: 1px solid rgba(128, 128, 128, 0.74);
+  border-bottom: 1px solid rgba(128, 128, 128, 0.74);
+  width: 48%;
+  height: 0.5vw;
+}
+
+.versus-icon {
+  background-color: black;
+  width: 4%;
+  height: 3.5vw;
+  border-radius: 50%;
+}
+
+.info-debate1,
+.info-debate3,
+.info-option1 {
+  width: 90%;
+}
+
+.debate-item1 {
+  padding-top: 2vw;
+  border-top: 3px solid #f72210;
+}
+
+.debate-item3 {
+  padding-bottom: 2vw;
+  border-bottom: 3px solid #1c87fb;
+}
+
+.opinion-item {
+  border-bottom: 1px solid rgb(189, 186, 186);
+}
+
+.more-stories-item {
+  border-top: 1px solid rgb(189, 186, 186);
+  margin-bottom: 2vw;
+}
+
+.info-more-stories1 {
+  width: 90%;
+}
+
+.it1 {
+  background-image: url("https://d.newsweek.com/en/full/1597646/tsunami-family-brothers.jpg?w=397&h=397&f=178afe1ac0a38a65047401f0ccbb1787");
+}
+
+.it2 {
+  background-image: url("https://d.newsweek.com/en/full/1596881/meghan-markle-look-alike-new-jersey.png?w=397&h=397&f=f991f48848d948d0de8bbdb99fed81f5");
+}
+
+.imc1 {
+  background-image: url("https://d.newsweek.com/en/full/1596461/30-black-owned-wineries-worth-supporting-right-now.jpg?w=397&h=397&f=3b36a58ac42aafdfe03b1afa325b2afb");
+}
+
+.imc2 {
+  background-image: url("https://d.newsweek.com/en/full/1596730/washington-dc-protest.jpg?w=397&h=397&f=21a0453c6dfd82377b0d78b1d6f6a73c");
+}
+
+.newsletter-signup-container {
+  width: 100%;
+  background-color: #fff;
+}
+
+.news-signup {
+  max-width: 22em;
+  background-color: #f4f5f7;
+  border: 1px solid #e5e5e5;
+}
+
+.magazine,
+.editors-pick,
+.picks-container {
+  background-color: #fff;
+}
+
+.nw-ti,
+.magazine-title-inside,
+.editors-title-inside {
+  font-size: 1.188;
+  font-weight: bolder;
+  color: black;
+}
+
+.icon-newsletter {
+  width: 47px;
+  height: 28px;
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+}
+
+.sign-email {
+  width: 100%;
+  height: 32px;
+  color: #555;
+  font-size: 14px;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: white;
+  border: 1px solid #e0e1e1;
+  text-indent: 9px;
+}
+
+.freesign-button {
+  width: 100%;
+  height: 32px;
+  color: white;
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 0;
+  font-family: "Roboto Condensed", sans-serif;
+  background: url(https://g.newsweek.com/www/images/free-sign-up.svg) center/108px no-repeat #f61602;
+  border-radius: 3px;
+  border: none;
+}
+
+.title-magazine,
+.title-editors {
+  width: 100%;
+}
```

Comment:
This could be achieved by the `w-100` class that Bootstrap provides. Remember to use Bootstrap classes wherever possible.

Reply:
thankyou I didn't know it